python/paddle/distributed/passes/__init__.py

```diff
@@ -18,6 +18,7 @@
 from .auto_parallel_sharding import *  # noqa: F403
 from .auto_parallel_amp import *  # noqa: F403
 from .auto_parallel_fp16 import *  # noqa: F403
+from .auto_parallel_bf16 import BF16Pass
```

Comment:
`from .auto_parallel_bf16 import *`

Reply:
If I use "from .auto_parallel_bf16 import *",I can not pass pre-commit.